src/useradd.c

```diff
@@ -749,6 +749,7 @@ static int set_defaults (void)
 	if (prefix[0]) {
 		free(default_file);
 	}
+	fclose (ofp);
 
 	return ret;
```

Comment:
This is double-closing in some cases.  The first close is in line 705.

Reply:
Surrounded with a condition to make sure that it's NULL.